tree/dataframe/src/RDataFrame.cxx

```diff
@@ -629,6 +629,31 @@ df = RDataFrame("mytree", "myfile.root", sparkcontext = sc)
 If an instance of [SparkContext](https://spark.apache.org/docs/latest/api/python/reference/api/pyspark.SparkContext.html)
 is not provided, the default behaviour is to create one in the background for you.
 
+### Distributed Snapshot
+
+The Snapshot operation behaves slightly differently when executed distributedly. First off, it requires the path
+supplied to the Snapshot call to be accessible from any worker of the cluster. Then, each distributed task will make
+a Snapshot on the range of entries it has been assigned with by the distributed RDataFrame application. Let's suppose
+our dataset is made of 100 entries. The default number of partitions in which the data will be split for distributed
+execution is 2. This means that a Snapshot operation on this dataset will produce two separate files, each holding the
+partial dataset resulting from applying the RDataFrame computation graph on either of the two ranges of entries.
+For example, the following application:
+
+~~~{.py}
+import ROOT
+RDataFrame = ROOT.RDF.Experimental.Distributed.Spark.RDataFrame
+df = RDataFrame(100)
+# Define a single column of 100 consecutive numbers
+df_x = df.Define("x","rdfentry_")
+# Snapshot the dataset distributedly
+snapdf = df_x.Snapshot("mytree","myfile.root")
+~~~
+
+Will result in two snapshotted files, respectively `myfile_0_49.root` and `myfile_50_99.root` where the numbers appended
```

Comment:
Doesn't come to mind anything obvious, other than creating a unique id for each partial snapshot. If we want to have a more "finalised" naming scheme already in this piece of documentation, we should first discuss it

Reply:
using a proper UUID is probably not great because then if you run the workflow twice you get two sets of output files, each file with its own unique name, and it's not trivial to separate the outputs of each run. just appending 1/2/3/4 would probably be ideal? what do you think?